### PR TITLE
#449 - Added ArtipieServer class

### PR DESCRIPTION
**Pull request by olegmoz**
Part of #449 
Added `ArtipieServer` class to run server in tests with single repo configured

### CODE REVIEW

File: src/test/java/com/artipie/ArtipieServer.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+        final Path repos = this.root.resolve("repos");
+        repos.toFile().mkdir();
+        Files.write(
+            repos.resolve(String.format("%s.yaml", this.name)),
+            this.config.getBytes()
+        );
+        final Path cfg = this.root.resolve("artipie.yaml");
+        Files.write(
+            cfg,
+            Yaml.createYamlMappingBuilder().add(
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/ArtipieServer.java:94]: @olegmoz we have code to create repository yaml config in several tests (in `YamlSettingsTest` for example), what do you think of adding todo to introduce class in test scope to create different configuration?

**Review by olenagerasimova — COMMENTED** [review]
@olegmoz one suggestion, please, take a look

File: src/test/java/com/artipie/ArtipieServer.java (hunk 1 above)
Comment (olegmoz) [reply src/test/java/com/artipie/ArtipieServer.java:101]: @olenagerasimova thanks for the suggestion, added a puzzle about it

**Review by olenagerasimova — APPROVED** [review]
@olegmoz thanks, looks good to me

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#468` is not assigned, can't get performer

**Pull request merged**